src/wazuh_db/schema_upgrade_v8.sql

```diff
@@ -10,6 +10,7 @@
 
 ALTER TABLE sys_osinfo ADD COLUMN reference TEXT NOT NULL DEFAULT '';
 ALTER TABLE sys_osinfo ADD COLUMN triaged INTEGER(1) DEFAULT 0;
+ALTER TABLE sync_info ADD COLUMN checksum TEXT NOT NULL DEFAULT '';
```

Comment:
This declaration is not the same than ` checksum TEXT NOT NULL CHECK (checksum <> '')`. Both clean installation and upgrade should have the same table structure

Reply:
Fixed